src/github.com/getlantern/autoupdate-server/server/github.go

```diff
@@ -272,12 +297,16 @@ func (g *ReleaseManager) pushAsset(os string, arch string, asset *Asset) (err er
 	if g.latestAssetsMap[os] == nil {
 		g.latestAssetsMap[os] = make(map[string]*Asset)
 	}
-	if g.latestAssetsMap[os][arch] == nil {
-		g.latestAssetsMap[os][arch] = asset
-	} else {
-		// Compare against already set version
-		if asset.v.GT(g.latestAssetsMap[os][arch].v) {
+
+	// Only considering non-manoto versions for the latestAssetsMap
+	if !buildStringContainsManoto(asset.v) {
```

Comment:
@myleshorton I was wrong with the `2.0.0+manoto` < `2.0.0+stable` assumption, actually `2.0.0+manoto` == `2.0.0+stable`. This change ignores all build with `+manoto` when looking for the most recent released version, so normal betas won't get updated to manoto versions.